clients/src/main/java/org/apache/kafka/clients/producer/ProducerRecord.java

```diff
@@ -19,6 +19,21 @@
  * If a valid partition number is specified that partition will be used when sending the record. If no partition is
  * specified but a key is present a partition will be chosen using a hash of the key. If neither key nor partition is
  * present a partition will be assigned in a round-robin fashion.
+ * <p>
+ * The record also has an associated timestamp. If user did not provide a timestamp, the producer will stamp the record
+ * with its current time. The timestamp eventually used by Kafka depends on the timestam
+ * <li>
+ * If the topic is configured to use {@link org.apache.kafka.common.record.TimestampType#CreateTime CreateTime}
```

Comment:
It seems that if we do not have the CreatTime at the end, java doc will show the entire qualifier, i.e. org.apache.kafka.common.record.TimestampType#CreateTime. This works but seems a little verbose. Alternatively we can import the TimestampType class so we don't need to have a display name for the ```@link```.

Do we have a convention for Java doc in Kafka? It seems we have different styles in the code base.